src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic/VisualBasicExtensionServiceProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.DotNet.UpgradeAssistant.Dependencies;
+using Microsoft.DotNet.UpgradeAssistant.Extensions;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic
+{
+    public class VisualBasicExtensionServiceProvider : IExtensionServiceProvider
+    {
+        public void AddServices(IExtensionServiceCollection services)
+        {
+            if (services is null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.Services.AddUpgradeStep<VisualBasicProjectUpdaterStep>();
```

Comment:
Do you need to register EnableMyDotSupportSubStep?

Reply:
Probably not since the VisualBasicProjectUpdaterStep will create its own substeps

Reply:
The VisualBasicProjectUpdaterStep is building this step directly currently. I saw that our other substep approaches both follow a ceremony that appeared to be unnecessary here.

Rather than having a list of substeps, the `ConfigUpdaterStep` gets a list of `IUpdater<ConfigFile>` and then uses a Substep as a proxy to invoke those objects. I'm currently skipping the proxy and just making the substep tightly coupled for now. I could change that, but it seemed to be mostly ceremony for now. I'm not even using DI in the substep yet.